src/diagnostics/core_diagnostics.jl

```diff
@@ -733,3 +733,315 @@ add_diagnostic_variable!(
+            out,
+            state.c.ρ .* cache.precomputed.cloud_diagnostics_tuple.q_liq,
+        )
+        return out
+    else
+        Operators.column_integral_definite!(
+            out,
+            state.c.ρ .* cache.precomputed.cloud_diagnostics_tuple.q_liq,
+        )
+    end
+end
+
+add_diagnostic_variable!(
+    short_name = "clwvi",
+    long_name = "Condensed Water Path",
+    standard_name = "atmosphere_mass_content_of_cloud_condensed_water",
+    units = "kg m-2",
+    comments = """
+    Mass of condensed (liquid + ice) water in the column divided by the area of the column 
+    (not just the area of the cloudy portion of the column). Includes precipitating hydrometeors
+    ONLY if the precipitating hydrometeors affect the calculation of radiative transfer in model.
+    """,
+    compute! = compute_clwvi!,
+)
+
+###
+# Dry Static Energy (1d)
+###
+compute_dsevi!(out, state, cache, time) =
+    compute_dsevi!(out, state, cache, time, cache.atmos.moisture_model)
+compute_dsevi!(_, _, _, _, model::T) where {T} =
+    error_diagnostic_variable("dsevi", model)
+
+function compute_dsevi!(
+    out,
+    state,
+    cache,
+    time,
+    moisture_model::T,
+) where {T <: Union{EquilMoistModel, NonEquilMoistModel}}
+    thermo_params = CAP.thermodynamics_params(cache.params)
+    if isnothing(out)
+        out = zeros(axes(Fields.level(state.f, half)))
+
+        cp = CAP.cp_d(cache.params)
+        z = Fields.coordinate_field(state.c.ρ).z
+        g = CAP.grav(cache.params)
+        Operators.column_integral_definite!(
+            out,
+            LB.@lazy_broadcasted @. (
+                state.c.ρ * (
+                    cp *
+                    TD.air_temperature(thermo_params, cache.precomputed.ᶜts) +
+                    g * z
+                )
+            )
+        )
+        return out
+    else
+        cp = CAP.cp_d(cache.params)
+        z = Fields.coordinate_field(state.c.ρ).z
+        g = CAP.grav(cache.params)
+
+        Operators.column_integral_definite!(
+            out,
+            LB.@lazy_broadcasted @. (
+                state.c.ρ * (
+                    cp *
+                    TD.air_temperature(thermo_params, cache.precomputed.ᶜts) +
+                    g * z
+                )
+            )
+        )
+    end
+end
+########### Temporary fix ##############
+import ClimaCore: Operators, RecursiveApply, Fields
+function Operators.column_integral_definite!(
+    ∫field,
+    ᶜinput,
+    init = RecursiveApply.rzero(eltype(∫field)),
+)
+    ᶜinput_times_Δz = Base.Broadcast.broadcasted(
+        RecursiveApply.:⊠,
+        ᶜinput,
+        Fields.Δz_field(axes(ᶜinput)),
+    )
+    Operators.column_reduce!(RecursiveApply.:⊞, ∫field, ᶜinput_times_Δz; init)
+end
+########### Temporary fix #############
+
+add_diagnostic_variable!(
+    short_name = "dsevi",
+    long_name = "Dry Static Energy Vertical Integral",
+    standard_name = "dry_static_energy_vertical_integral",
```

Comment:
```suggestion
```
(same for other variables, if there is no `standard_name` from the cmip table, just remove the `standard_name`)